### PR TITLE
Chloe feature post presentation

### PR DESCRIPTION
**Pull request by Chloe-Meinshausen**
Hey guys,
   I have cleaned up and removed console logs and some components
   I also adjusted the font on my rentals list so that it shrinks on smaller screens
   I fixed a bug that stopped the user's profile from loading from the nav bar if he was already visiting someone else's profile
   adjusted the sizing on a few pages so they look better on mobile phones

### CODE REVIEW

File: client/src/account/Profile.jsx [hunk 1]
```diff
@@ -15,25 +15,30 @@ import { useUserAuth } from '../context/UserAuthContext.jsx';
...
-
-  //get id from url
   let { id } = useParams();
   id = isNaN(id) ? id === 'undefined' ? localId : id.substring(3) : id;
+  let [curId, setCurId] = useState(id);
+  //get id from url
+
 
   useEffect(()=> {
+    console.log('call');
```
Comment (rudyesar) [inline client/src/account/Profile.jsx]: I think you may have missed this console.log here

File: client/src/account/ProfileTabs.jsx [hunk 2]
```diff
@@ -4,7 +4,6 @@ import React, {useState} from 'react';
 import TabContent from 'react-bootstrap/TabContent';
 import {Link } from 'react-router-dom';
```
Comment (rudyesar) [inline client/src/account/ProfileTabs.jsx]: Check the spacing inside the curly braces here

File: client/src/account/RentalTabs.jsx [hunk 3]
```diff
@@ -4,7 +4,6 @@ import React, {useState} from 'react';
 import TabContent from 'react-bootstrap/TabContent';
 import {Link } from 'react-router-dom';
```
Comment (rudyesar) [inline client/src/account/RentalTabs.jsx]: spacing

File: client/src/account/accountStyles.css [hunk 4]
```diff
@@ -55,3 +65,26 @@ a {
...
+  h1 {font-size:1.25rem; }
+}
+
+@media (min-width: 950px) {
+  h4 {font-size:1rem;}
+  h3 {font-size:1.2rem;}
+  h2 {font-size:1.2rem;}
+  h1 {font-size:1.3rem;}
+
+}
```
Comment (rudyesar) [inline client/src/account/accountStyles.css:90]: if you have time, I'd suggest just reviewing the spacing before and after many curly braces in this file to make it a bit more consistent.

**Review by rudyesar — COMMENTED** [review]
Left a couple small comments throughout the PR

File: client/src/account/ProfileTabs.jsx (hunk 2 above)
Comment (Chloe-Meinshausen) [reply client/src/account/ProfileTabs.jsx]: Thank you!

**Comment by Chloe-Meinshausen** [comment]
Thanks a bunch, Rudy. I think I fixed all spacing errors you mention plus a few more!

**Pull request merged**